Description: Correct pod NAME issue in LedgerSMB/IC.pm
 Is missing a pod NAME entry altogether.
Bug: https://sourceforge.net/tracker/?func=detail&aid=3472601&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/IC.pm
+++ b/LedgerSMB/IC.pm
@@ -1,3 +1,11 @@
+=pod
+
+=head1 NAME
+
+LedgerSMB::IC - Inventory Control backend
+
+=cut
+
 #=====================================================================
 # LedgerSMB
 # Small Medium Business Accounting software
